reporter/common.go

```diff
@@ -0,0 +1,39 @@
+package reporter
+
+import (
+	"fmt"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/wavefronthq/wavefront-opentracing-sdk-go/tracer"
+	wf "github.com/wavefronthq/wavefront-sdk-go/senders"
+)
+
+func prepareReferences(span tracer.RawSpan) ([]string, []string) {
+	parents := make([]string, 0)
+	followsFrom := make([]string, 0)
```

Comment:
Use `var parents []string` and `var followsFrom []string` instead. 
Using make allocates memory and majority of spans will have a single parent or followsFrom.